plugins/net_plugin/net_plugin.cpp

```diff
@@ -2071,17 +2029,13 @@ namespace eosio {
    }
 
    void dispatch_manager::bcast_transaction(const packed_transaction& trx) {
-      const auto& id = trx.id();
-      time_point_sec trx_expiration = trx.expiration();
-      node_transaction_state nts = {id, trx_expiration, 0, 0};
-
       std::shared_ptr<std::vector<char>> send_buffer;
-      for_each_connection( [this, &trx, &nts, &send_buffer]( auto& cp ) {
+      auto now = fc::time_point::now();
```

Comment:
make this const to make it clear that now is the same for all connections.

Reply:
Done